Eldora Scraper Is Broken in Production

Seeing the following error in productions logs:
`ubuntu@ski-resort-dashboard:~$ 2023-01-19T18:02:10.150710052Z app[web.1]: play.api.UnexpectedException: Unexpected exception[Exception: Resort Eldora-Mountain-Resortwasn't added to the database snapshots Map for storage. Something maybe wrong with the data scraping for said resort]
2023-01-19T18:02:10.150713879Z app[web.1]:      at play.api.http.HttpErrorHandlerExceptions$.throwableToUsefulException(HttpErrorHandler.scala:358)
2023-01-19T18:02:10.150717315Z app[web.1]:      at play.api.http.DefaultHttpErrorHandler.onServerError(HttpErrorHandler.scala:264)
2023-01-19T18:02:10.150720621Z app[web.1]:      at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:430)
2023-01-19T18:02:10.150723837Z app[web.1]:      at play.core.server.AkkaHttpServer$$anonfun$2.applyOrElse(AkkaHttpServer.scala:422)
2023-01-19T18:02:10.150726993Z app[web.1]:      at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:490)
2023-01-19T18:02:10.150730109Z app[web.1]:      at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:63)
2023-01-19T18:02:10.150733245Z app[web.1]:      at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:100)
2023-01-19T18:02:10.150736551Z app[web.1]:      at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.scala:18)
2023-01-19T18:02:10.150739847Z app[web.1]:      at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:94)
2023-01-19T18:02:10.150742913Z app[web.1]:      at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:100)
2023-01-19T18:02:10.150746029Z app[web.1]:      at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:49)
2023-01-19T18:02:10.150749105Z app[web.1]:      at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:48)
2023-01-19T18:02:10.150752251Z app[web.1]:      at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
2023-01-19T18:02:10.150755306Z app[web.1]:      at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
2023-01-19T18:02:10.150758383Z app[web.1]:      at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
2023-01-19T18:02:10.150761459Z app[web.1]:      at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:175)
2023-01-19T18:02:10.150764825Z app[web.1]: Caused by: java.lang.Exception: Resort Eldora-Mountain-Resortwasn't added to the database snapshots Map for storage. Something maybe wrong with the data scraping for said resort
2023-01-19T18:02:10.150779002Z app[web.1]:      at dao.ResortData.getResortSnapshot(ResortData.scala:86)
2023-01-19T18:02:10.150782408Z app[web.1]:      at dao.ResortData.setSnapshotForResort(ResortData.scala:71)
2023-01-19T18:02:10.150786997Z app[web.1]:      at controllers.HomeController.$anonfun$scrape$2(HomeController.scala:59)
2023-01-19T18:02:10.150790203Z app[web.1]:      at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:467)
2023-01-19T18:02:10.150793359Z app[web.1]:      at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1402)
2023-01-19T18:02:10.150796755Z app[web.1]:      ... 4 common frames
`

--- a/app/models/WebModel.scala
+++ b/app/models/WebModel.scala
@@ -109,14 +109,14 @@ case object Keystone extends Resorts {
 case object Eldora extends PowdrResorts {
     override def toString: String = "Eldora-Mountain-Resort"
     override val databaseName: String = "ELDORA"
-    override val scrapeUrl: String = "https://www.eldora.com/api/v1/dor/conditions"
+    override val scrapeUrl: String = "https://api.eldora.com/api/v1/dor/conditions"
     override val location_id: Int = 11
 }
 
 case object Copper extends PowdrResorts {
     override def toString: String = "Copper-Mountain"
     override val databaseName: String = "COPPER"
-    override val scrapeUrl: String = "https://www.coppercolorado.com/api/v1/dor/conditions"
+    override val scrapeUrl: String = "https://api.coppercolorado.com/api/v1/dor/conditions"
     override val location_id: Int = 7
 }
 

--- a/app/scrapers/PowdrScraper.scala
+++ b/app/scrapers/PowdrScraper.scala
@@ -19,15 +19,17 @@ class PowdrScraper (ws: WSClient, resort: PowdrResorts)(
 
     private val request = ws.url(resort.scrapeUrl)
     private val snowReportResult: SnowResult = Await.result(request.get().map { response =>
-        (response.json \ "snowReport1234").validate[Set[SnowResult]]
+        (response.json \ "snowReport").validate[Set[SnowResult]]
     }, 5.seconds).getOrElse(Set(defaultResult))
         .find(report => report.location_id == resort.location_id).getOrElse(defaultResult)
 
     override protected def scrape24HrSnowFall(): Int = {
+        print(snowReportResult.items)
         snowReportResult.items.find(item => item.duration == "24 Hours").getOrElse(SnowAmount(0,"")).amount
     }
     
     override protected def scrapeBaseDepth(): Int = {
+        print(snowReportResult.items)
         snowReportResult.items.find(item => item.duration == "base-depth").getOrElse(SnowAmount(0,"")).amount
     }
     

